src/Symfony/Bridge/Twig/Tests/Extension/EmojiExtensionTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Twig\Tests\Extension;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Bridge\Twig\Extension\EmojiExtension;
+
+class EmojiExtensionTest extends TestCase
+{
+    /**
+     * @testWith ["🅰️", ":a:"]
+     *           ["🅰️", ":a:", "slack"]
+     *           ["🅰", ":a:", "github"]
+     */
+    public function testEmojify(string $expected, string $string, ?string $catalog = null)
```

Comment:
This test is failing in the appveyor build. It should be skipped when we don't have ext-intl

Reply:
Thanks, fixed